tests/unit/SubmittedFormExtensionTest.php

```diff
@@ -76,26 +75,6 @@ public function testUpdateAfterProcessSession()
         $this->assertNull($session);
     }
 
-    public function testIsPartial()
```

Comment:
Why did you remove these?

Reply:
I removed `IsPartial` because I don't see the need for it. Data from `PartialFormSubmission` is already partial.

Reply:
@Firesphere Finally, tests are green. Are there any changes you want me to address?